scripts/default/opensearch-dashboards/build.sh

```diff
@@ -98,7 +98,7 @@ mkdir -p $OUTPUT/plugins
 cp -r ../$PLUGIN_NAME/ ../OpenSearch-Dashboards/plugins
 echo "BUILD MODULES FOR $PLUGIN_NAME"
 CURR_DIR=`pwd`
-cd ../OpenSearch-Dashboards; eval $NVM_CMD; yarn osd bootstrap
+cd ../OpenSearch-Dashboards; eval $NVM_CMD; yarn osd bootstrap --single-version=loose
```

Comment:
This is not fully applicable due to this will affect 1.x.

Reply:
@peterzhuamazon what do you suggest here, should we add a version check here to make sure we only run this above a specific version?

Reply:
@peterzhuamazon Any idea how to proceed here? Looks like many plugins are failing for 2.13.0 because of this.